job/common/src/main/java/alluxio/job/wire/JobWorkerHealth.java

```diff
@@ -35,10 +36,11 @@
    * @param loadAverage output of CentralProcessor.getSystemLoadAverage on the worker
    * @param hostname hostname of the worker
    */
-  public JobWorkerHealth(long workerId, double[] loadAverage, String hostname) {
+  public JobWorkerHealth(long workerId, double[] loadAverage, int taskPoolSize, String hostname) {
```

Comment:
NIT: update the javadoc for this parameter

Reply:
This should have been fixed.